containers/docker/README.md

```diff
@@ -61,16 +40,36 @@ Providing a PostgreSQL password on the command line such as in the preceeding is
 See [this section on Docker Secrets](https://github.com/docker-library/docs/blob/master/postgres/README.md#docker-secrets) on how to avoid passing credentials on the command line.  Further, also [consider this example](https://github.com/pglombardo/PasswordPusher/issues/412) provided by [Viajaz](https://github.com/Viajaz).
 
 
-## pwpush-mysql
+## mysql
 
-This container uses a default database URL of:
+To setup the container to use the mysql/mariadb database backend DATABASE_URL environment variable needs to be configured. The syntax should look like:
 
     DATABASE_URL=mysql2://passwordpusher_user:passwordpusher_passwd@mysql:3306/passwordpusher_db
 
-You can either configure your MySQL server to use these credentials or override the environment var in the command line:
+You can either configure your MySQL / MariaDB server to use these credentials or override the environment var in the command line:
+
+    docker run -d -p "5100:5100" -e "DATABASE_URL=mysql2://pwpush_user:pwpush_passwd@mysql:3306/pwpush_db" pglombardo/pwpush:latest
+
+_Note: Providing a MySQL / MariaDB password on the command line is far less than ideal_
+
+
+## Tags
+
+| Tag Name | Description |
+|-|-|
+| `latest` | Builds off of the latest code.  May occasionally be unstable. |
+| `release` | Points to the latest _stable_ release. |
+| `X.X.X` | Semantic version tags. |
+
+When in doubt, use `release`.
 
-    docker run -d -p "5100:5100" -e "DATABASE_URL=mysql2://pwpush_user:pwpush_passwd@mysql:3306/pwpush_db" pglombardo/pwpush-mysql:latest
+`amd64` and `arm64` architectures are both built.  Note [this bug](https://github.com/pglombardo/PasswordPusher/issues/268) in regards to tag availability for the `arm64` architecture.
```

Comment:
Is this still true @pglombardo? The obsolete images on docker hub do have `linux\amd64` and `linux\arm64`

Reply:
Good point - that is no longer a bug.  Previously the `release` tag would only point to one architecture.